### ISSUE TITLE
Transfer history empty if one payment is not "Payment Processed" status

### ISSUE THREAD

**Issue by arrenv** [JoinColony/colonyCDapp#2962]
## Steps to reproduce

Not sure why this would be the case, but it was what I experienced testing the feature, it could have been just a coincidence but seemed to line up exactly with the status. 

1. Create a crypto to fiat payment that is successful.
2. Check the Transfer history table.
3. Note that the payment should appear.
4. Create a crypto to fiat payment that is still pending.
5. Check the Transfer history table.
6. Note that no entries show at all.
7. Check the Transfer history table after the new payment is successful.
8. Note the Transfer history table should show all correct data.

## Expected behaviour

* It should show all user's transfers, regardless of the status of the payment.
* If there is a new payment that has not succeeded yet, you should still be able to see all past transfers.

## Actual behaviour

* It seems that if you have a payment that has not succeeded yet, then none of the Transfer history appears until the new payment succeeds.

#### While new payment was pending

![image](https://github.com/user-attachments/assets/e1ae6356-b0b5-473a-ba01-2dc12b6dcb5d)

![image](https://github.com/user-attachments/assets/f8b5a2c5-c0ce-42d3-9c5e-8282e19d22f0)

#### After new payment succeeded

![image](https://github.com/user-attachments/assets/9423ad53-8839-4aa8-84a2-375b772f9e8c)

![image](https://github.com/user-attachments/assets/69413692-cfe5-4b1d-a4cb-30b174998a4f)

**Comment by rdig** [comment]
Yup this is a very valid issue, I just want to note, out of personal testing experience, that transfers don't show up immediately on the bridge side, even in "pending states".

It takes a good while until you see "payment submitted" payments in their API
